lib/data/repositories/database_repository_impl.dart

```diff
@@ -101,4 +107,16 @@ class DatabaseRepositoryImpl implements DatabaseRepository {
       return const Left(Failure(error: Errors.unknownError));
     }
   }
+
+  @override
+  Future<Either<Failure, Success>> updateUserProfile(UserProfileEntity entity)async  {
+    try{
+      await _remoteSource.updateUserProfile(UserProfileDto.fromEntity(entity));
+      return const Right(Success());
+    } on ApiException catch (e) {
+      return Left(Failure(error: e.failure));
+    }catch (e) {
```

Comment:
You need to enable auto format or config git hooks

Reply:
i don't understand